irods/message/__init__.py

```diff
@@ -652,32 +652,47 @@ class ObjCopyRequest(Message):
     srcDataObjInp_PI = SubmessageProperty(FileOpenRequest)
     destDataObjInp_PI = SubmessageProperty(FileOpenRequest)
 
+#in iRODS <= 4.2.11:
 # define ModAVUMetadataInp_PI "str *arg0; str *arg1; str *arg2; str *arg3;
 # str *arg4; str *arg5; str *arg6; str *arg7;  str *arg8;  str *arg9;"
 
+#in iRODS > 4.2.11:
+# define ModAVUMetadataInp_PI "str *arg0; str *arg1; str *arg2; str *arg3;
+# str *arg4; str *arg5; str *arg6; str *arg7;  str *arg8;  str *arg9; struct KeyValPair_PI"
```

Comment:
What happens if you just send KeyValPair_PI indiscriminately? Will older servers just ignore everything after arg9?

Reply:
leave as is. made new issue to remove dynamic class generation